sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Helpers/ContainerRepository.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Containers.ContainerRegistry
+{
+    /// <summary> A helper class that groups information and operations about a repository in this container registry. </summary>
+    public partial class ContainerRepository
+    {
+        private readonly ClientDiagnostics _clientDiagnostics;
+        private readonly ContainerRegistryRestClient _restClient;
```

Comment:
As I suspect from the method signatures above: this is a client and, therefore, should end in "Client".

Reply:
This is not a client - see comment above.